### PR TITLE
Year 2 extension

### PR DESCRIPTION
**Pull request by asegun-cod**
- Add version 5 of the data categorisation framework
- Add sub-category trend tab 
- Replace ordinary HTML text output with data tables and add a download button to each table
- Add information to the header and link to contact the project team
- refactor comment search to use regex 
- refactor date filter to be derived from database data
- add Github Actions for deployment to Connect server and to do R-cmd check
- add some unit test 
- add configuration settings for 4th pilot trust
- add functionality to view inter-relationship within sub-categories of a category or across all categories

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,67 +1,66 @@
 Package: experiencesdashboard
 Title: Patient Experiences Dashboard
 Version: 0.4.0
```
Comment (ChrisBeeley) [inline DESCRIPTION:3]: Bump version

File: R/app_server.R [hunk 2]
```diff
@@ -5,39 +5,37 @@
...
     dplyr::tally() %>%
     dplyr::pull(n) > 0
-  
-  if (!data_exists){
+
+  if (!data_exists) {
     showModal(modalDialog(
-      title = "NO DATA!",
-      HTML("<strong>Please go to the 'Data Management' tab to upload data.</strong>"),
+      title = strong("WELCOME!!!"),
```
Comment (ChrisBeeley) [inline R/app_server.R:30]: Can we change this to "Welcome!" instead?

**Review by ChrisBeeley — CHANGES_REQUESTED** [review]
Minor updates

File: DESCRIPTION (hunk 1 above)
Comment (asegun-cod) [reply DESCRIPTION:3]: what will the version be: `0.7.0` (inline with next release tag) or `0.5.0`??

File: R/app_server.R (hunk 2 above)
Comment (asegun-cod) [reply R/app_server.R:30]: should we add the Trust name here e.g
Welcome to Nottinghamshire NHS Trust, Patient Experience Qualitative Data Categorisation Dashboard

Comment (ChrisBeeley) [reply R/app_server.R:30]: I'd avoid. Sometimes orgs change their name and it could get fiddly

**Review by ChrisBeeley — APPROVED** [review]

**Pull request merged**